pkg/skaffold/schema/v1alpha5/upgrade_test.go

```diff
@@ -116,7 +116,7 @@ profiles:
 	verifyUpgrade(t, yaml, expected)
 }
 
-func upgradeShouldFailt(t *testing.T, input string) {
+func upgradeShouldFail(t *testing.T, input string) {
```

Comment:
It's hard to tell if a `t` here is meaningful. If it is, I'd hope you'd tolerate `upgradeShouldFailT` as opposed to the current.